src/QandaAtom.tsx

```diff
@@ -1,7 +1,450 @@
+import { headline, textSans, body } from '@guardian/src-foundations/typography';
+import { SvgPlus, SvgMinus } from '@guardian/src-icons';
+
+// CSS
+const containerStyling = css`
+    display: block;
+    position: relative;
+    margin-bottom: 0.75rem;
+    margin-top: 1rem;
+`;
+
+const detailStyling = css`
+    margin: 16px 0 36px;
+    background: ${neutral[93]};
+    padding: 0 5px 6px;
+    border-image: repeating-linear-gradient(
+            to bottom,
+            ${neutral[86]},
+            ${neutral[86]} 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+    border-top: 13px solid black;
+    border-left: 0;
+    border-right: 0;
+    border-bottom: 0;
+    position: relative;
+    summary {
+        list-style: none;
+        margin: 0 0 16px;
+    }
+    summary::-webkit-details-marker {
+        display: none;
+    }
+
+    summary:focus {
+        outline: none;
+    }
+`;
+
+const bodyStyling = css`
+    ${body.medium()}
+    p {
+        margin-bottom: 0.5rem;
+    }
+    @media (min-width: 46.25em) {
+        ul {
+            list-style: none;
+            margin-bottom: 1rem;
+        }
+    }
+
+    ul {
+        list-style: none;
+        margin: 0 0 0.75rem;
+        padding: 0;
+    }
+
+    li {
+        margin-bottom: 0.375rem;
+        padding-left: 1.25rem;
+    }
+
+    li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.375rem;
+        height: 0.75rem;
+        width: 0.75rem;
+        margin-right: 0.5rem;
+        background-color: ${neutral[86]};
+        margin-left: -1.25rem;
+    }
+
+    a {
+        color: #ab0613;
+        text-decoration: none !important;
+        border-bottom: 0.0625rem solid #bdbdbd;
+        transition: border-color 0.15s ease-out;
+    }
+
+    a:hover {
+        border-bottom: solid 0.0625rem ${news[400]};
+    }
+
+    b {
+        font-weight: bold;
+    }
+`;
+
+const summaryStyling = css``;
+
+const showHideStyling = css`
+    background: ${neutral[7]};
+    color: ${neutral[100]};
+    height: 2rem;
+    font-size: 13px;
+    position: absolute;
+    bottom: 0;
+    transform: translate(0, 50%);
+    padding: 0 15px 0 7px;
+    border-radius: 100em;
+    cursor: pointer;
+    display: inline-flex;
+    align-items: center;
+    border: 0;
+    margin: 0;
+    :hover {
+        background: #e00000;
+    }
+`;
+
+const buttonRound = css`
+    display: inline-flex;
+    cursor: pointer;
+    align-items: center;
+    justify-content: center;
+    background: black;
+    color: white;
+    border-style: hidden;
+    border-radius: 100%;
+    margin: 0 0 0 5px;
+    padding: 0;
+    width: 28px;
+    height: 28px;
+    :hover {
+        background: ${news[400]};
+        border-color: ${news[400]};
+    }
+
+    :focus {
+        border: none;
+    }
+`;
+
+const imageStyling = css`
+    float: left;
+    margin-right: 16px;
+    margin-bottom: 6px;
+    width: 100px !important;
```

Comment:
`!important` is often a code smell - what are you having to override here (and other usages)?